mlir/include/mlir-c/Dialect/LLVM.h

```diff
@@ -239,7 +239,9 @@ MLIR_CAPI_EXPORTED MlirAttribute mlirLLVMDICompositeTypeAttrGet(
     MlirContext ctx, unsigned int tag, MlirAttribute recId, MlirAttribute name,
     MlirAttribute file, uint32_t line, MlirAttribute scope,
     MlirAttribute baseType, int64_t flags, uint64_t sizeInBits,
-    uint64_t alignInBits, intptr_t nElements, MlirAttribute const *elements);
+    uint64_t alignInBits, intptr_t nElements, MlirAttribute const *elements,
+    MlirAttribute location, MlirAttribute rank, MlirAttribute allocated,
```

Comment:
```suggestion
    MlirAttribute dataLocation, MlirAttribute rank, MlirAttribute allocated,
```
nit: should it be dataLocation as well for consistency?